<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget{
border-radius:10px;
background-image: url(:/new/prefix1/background);
}
QToolButton#closeButton{
color:rgb(0,0,0);
border-style:none;
border-top-right-radius:10px;
}
QToolButton#closeButton::hover{
color:rgb(255,255,255);
background-color: rgba(255, 56, 56, 150);
border-top-right-radius:10px;
}
QToolButton#minimizeButton{
border-style:none;
color:rgb(0,0,0);
}
QToolButton#minimizeButton::hover{
border-style:none;
color:rgb(0,0,0);
background-color: rgba(203, 203, 203, 150);
}
QToolButton#checkButton{
border:2px solid;
border-color:rgb(200,200,200);
border-radius:15px;
}                                              
QToolButton#checkButton::hover{
background-color:rgb(230,230,230);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QToolButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QToolButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>800</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="check1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="check2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>**</string>
    </property>
   </widget>
   <widget class="QLabel" name="checkLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>400</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择乘方的表示形式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hello!</string>
    </property>
   </widget>
   <widget class="QLabel" name="clockLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>倒计时：20</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
